stages/stage_2/helpers.py

```diff
@@ -7,23 +7,23 @@
 
 def quote(sexp):
     """quoted list as a python list, not as an sexp"""
-    return [QUOTE_KW, sexp]
+    return (QUOTE_KW, sexp)
```

Comment:
Do I understand correctly that this could also be spelled: `quote(sexp)`?

Reply:
Well, this is the `quote(sexp)` definition :)

But otherwise, yes.